### DISCUSSION TITLE
Adapt/reduce reproducibility standards

### DISCUSSION THREAD

**Discussion by angaertner** [RESQUE-Framework/website#118] [Ideas]
Sufficiency - Soup-to-Nuts - Portability - Almost  --> reduce!

Last two questions are redundant. Either I can reproduce everything, then it probably should fulfill everything above. Also, I find the options here too detailed. If Portability is fulfilled, shouldn’t the soup-to-nuts automatically be part of this?

(Feedback Methoden)

**Comment by nicebread** [comment]
We removed it and added two aspects into another indicator.
